lib/ModelMixins/CatalogFunctionJobMixin.ts

```diff
@@ -201,6 +202,7 @@ function CatalogFunctionJobMixin<
      * - `pollForResults` returns true {@link CatalogFunctionJobMixin#refreshData}
      * - on `loadMetadata` if `jobStatus` is "finished", and `!downloadedResults`  {@link CatalogFunctionJobMixin#forceLoadMetadata}
      */
+    @action
```

Comment:
Sorry to be a pain @na9da can you please also change this async function to call runInActions where required

Reply:
actually scratch that - sorry I'm being too pedantic.